Practice_Django/shop/store/templates/store/base.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Django Store</title>
+</head>
+<body>
+    <h1><a href="/">Django Store</a></h1>
+    <hr>
+    {% block content %}
+    {% endblock %}
+</body>
+</html>
```

Comment:
A line blank at the end
```suggestion
</html>

```